src/app-layout/index.tsx

```diff
@@ -73,10 +76,31 @@ const AppLayout = React.forwardRef(
     const props = { contentType, headerSelector, footerSelector, ...rest, ariaLabels };
 
     const baseProps = getBaseProps(rest);
+    const ownDrawers = (props as any).drawers;
+    const disableRuntimeDrawers = (props as any).__disableRuntimeDrawers;
```

Comment:
[minor] interested in the difference between `drawers` and `__disableRuntimeDrawers` - is one "more" private than the other?

Reply:
Yes. `drawers` has potential to become an official public API. `__disableRuntimeDrawers` is not, it will be removed after runtime phase of this API ends